### ISSUE TITLE
Can I and how to provide new training data

### ISSUE THREAD

**Issue by Pazdikan** [Manuteaa/dbd_autoSkillCheck#13]
I love the recent updates and thanks for the work you put in this project 😊

I had a couple merciless storm games and some of the merciless storm skillchecks were not hit properly (hit one frame earlier, and the webstite shows 60/40 % split between two properties, I dont remember which ones). Sometimes hyperfocus with max stacks is getting hit too late resetting the stacks.

I know that the model will never be 100% perfect, but nonetheless as I play the game a lot, I'd like to help with gathering new images. The debug flag "Save all skill check frames in saved_images (will impact fps)" doesn't really impact the model's FPS that much, and everything works like it should. I've noticed that the folders are automatically numbered. Does it mean I can just zip the entire folder, and send it to you when I get enough data or does it still needs some manual review?

**Comment by Manuteaa** [comment]
Hello ! Thanks :)

Yes the model can't be 100% reliable 😮‍💨 

In fact, to enhance the performance of the model, we should :
- Collect more data, with more diversity : I have indeed implemented the option to automatically save the frames and automatically label them in the correct folders. Of course, it there was a skill check fail, some of the frames may not be labeled correctly, so you need to manually fix it by moving the frames in the correct folder, send send me the folders.
- Be more precise when we label the data (even in the data we have already collected and labeled, some of them are wrong I think, especially for labels 2&3. We need to check and fix our whole current dataset but it is soo tedious)
- Spend more time and money to train the models. There are many hyper-parameters to adjust when training the model, such as the parameters of the optimizer. Training the model is an iterative process which takes many hours, and it requires finding the best hyper-parameters combination. In addition, I can't run the training process all the time on my own laptop, so I trained the model using AWS and it cost me around 30$ to get this current model.

So, if you want to enhance the model, I can give you the first a second task :D Ready ?
For the third one I will see when I have time & money for this project.
